games-rpg/devilutionx_extras/devilutionx_extras-v4.recipe

```diff
@@ -0,0 +1,39 @@
+SUMMARY="Language specific content which is optional for running DevilutionX"
+DESCRIPTION="This package contains some language specific content for DevilutionX:\
+
+* CJK languages and fonts support
+* Spanish language dub
+* Polish language dub
+* Russian language dub"
+HOMEPAGE="https://devilutionx.com/"
+COPYRIGHT="2023-2024 DevilutionX"
+LICENSE="SIL Open Font License v1.1"
+REVISION="1"
+SOURCE_URI="https://github.com/diasurgical/devilutionx-assets/releases/download/$portVersion/fonts.mpq#noarchive"
+CHECKSUM_SHA256="551ecee2d95b4e7807737a7794a6bacf0b4a03a91634816277b91db35ce1e259"
+SOURCE_FILENAME="fonts.mpq"
```

Comment:
Usually, these should contain the portVersion to make future updates easier without having to manually delete them first (otherwise HaikuPorter may think that it already downloaded a file if one of the same name already exists), but as you are using these directly, I guess this is easier like that. I think that SOURCE_FILENAME is even redundant in this case then.